cloudify_cli/config/parser_config.py

```diff
@@ -166,6 +178,111 @@ def parser_config():
                     }
                 }
             },
+            'snapshots': {
+                'help': "Manages Cloudify's Snapshots",
+                'sub_commands': {
+                    'create': {
+                        'arguments': {
+                            '-s,--snapshot-id': argument_utils.remove_completer(
+                                snapshot_id_argument(
+                                    hlp='A unique id that will be assigned to the created snapshot'
+                                )
+                            ),
+                            '--include-metrics': {
+                                'dest': 'include_metrics',
+                                'action': 'store_true',
+                                'default': False,
+                                'help': 'Specify this flag to include metrics data'
+                                        'in the snapshot'
+                            },
+                            '--exclude-credentials': {
+                                'dest': 'exclude_credentials',
+                                'action': 'store_true',
+                                'default': False,
+                                'help': 'Specify this flag to prevent storing credentials'
+                                        'in the snapshot'
+                            }
+                        },
+                        'help': 'command for creating a new snapshots',
+                        'handler': cfy.snapshots.create
+                    },
+                    'upload': {
+                        'arguments': {
+                            '-p,--snapshot-path': {
+                                'metavar': 'SNAPSHOT_FILE',
+                                'dest': 'snapshot_path',
```

Comment:
do we intend to let users provide a URL instead of a local path for uploading and restoring snapshots?